### PR TITLE
rpk 24.2

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Updates rpk for 24.2
Review deadline: 9 July

### Changelog
New commands
[`rpk cluster quotas alter`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-cluster/rpk-cluster-quotas-alter/)
[`rpk cluster quotas describe`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-cluster/rpk-cluster-quotas-describe/)
[`rpk cluster quotas import`](https://deploy-preview-611--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-cluster/rpk-cluster-quotas-import/)
[`rpk cluster quotas`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-cluster/rpk-cluster-quotas/)
[`rpk transform pause`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-transform/rpk-transform-pause/)
[`rpk transform resume`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-transform/rpk-transform-resume/)

New flags or improved descriptions
[`rpk cluster partitions list`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-cluster/rpk-cluster-partitions-list/)
[`rpk cluster self-test start`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-cluster/rpk-cluster-self-test-start/)
[`rpk registry schema check combatibility`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-registry/rpk-registry-schema-check-compatibility/)
[`rpk registry schema create`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-registry/rpk-registry-schema-create/)
[`rpk registry schema get`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-registry/rpk-registry-schema-get/)
[`rpk transform deploy`](https://deploy-preview-591--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-transform/rpk-transform-deploy/)

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-list.adoc [hunk 1]
```diff
@@ -77,7 +43,9 @@ list, ls, describe
 
 |-h, --help |- |Help for list.
 
-|-p, --partition |- |ints   List of comma-separated partitions IDs that you wish to filter the results with.
+|-n, --node-ids |ints |List of comma-separated broker IDs that you wish to filter the results with.
+
+|-p, --partition |ints |List of comma-separated partitions IDs that you wish to filter the results with.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-list.adoc]:
```suggestion
|-n, --node-ids |ints |List of comma-separated broker IDs you wish to use to filter the results.

|-p, --partition |ints |List of comma-separated partitions IDs you wish to use to filter the results.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-list.adoc [hunk 2]
```diff
@@ -87,4 +55,42 @@ list, ls, describe
...
+----
+
+List all partitions in the cluster, filtering for topic foo and bar:
+
+[,bash]
+----
+rpk cluster partitions list foo bar
+----
+
+List partitions which replicas are assigned to brokers 1 and 2.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-list.adoc]:
```suggestion
List partitions with replicas that are assigned to brokers 1 and 2.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc [hunk 3]
```diff
@@ -0,0 +1,81 @@
+= rpk cluster quotas alter
+:description: rpk cluster quotas alter
+
+Add or delete a client quota.
+
+A client quota consists of an entity (to whom the quota is applied) and a quota type (what is being applied).
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc]:
```suggestion
A client quota consists of an entity (to which the quota is applied) and a quota type (what is being applied).
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+There are two entity types supported by Redpanda: client ID and client ID prefix. Assigning quotas to default entity types is possible using the `--default` flag.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc]:
```suggestion
There are two entity types supported by Redpanda: client ID and client ID prefix. Use the `--default` flag to assign quotas to default entity types.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+|--delete |strings |Key of the quota to delete (repeatable).
+
+|--dry |- |Key of the quota to delete (repeatable).
+
+|--format |string |Output format. Possible values: `json`, `yaml`, `text`, `wide`, `help`. Default: `text`.
+
+|-h, --help |- |Help for alter.
+
+|--name |strings |Entity for exact matching. Format type=name where type is client-id or client-id-prefix (repeatable).
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc]:
```suggestion
|--name |strings |Entity for exact matching. Format `type=name` where `type` is the `client-id` or `client-id-prefix` (repeatable).
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc]:
```suggestion
|--config |string |Redpanda or `rpk` configuration file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc [hunk 7]
```diff
@@ -0,0 +1,67 @@
+= rpk cluster quotas describe
+:description: rpk cluster quotas describe
+
+Describe client quotas.
+
+This command describes client quotas that match the provided filtering criteria. Running the command without filters returns all client quotas. Use the
+`--strict` flag for strict matching, which means that the only quotas returned  exactly match the filters.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc]:
```suggestion
This command describes client quotas that match the provided filtering criteria. Running the command without filters returns all client quotas. Use the
`--strict` flag for strict matching, which means that the only quotas returned exactly match the filters.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+Filters can be provided in terms of entities. An entity consists of either a client ID or a client ID prefix.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc]:
```suggestion
You can specify filters in terms of entities. An entity consists of either a client ID or a client ID prefix.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+rpk cluster quotas describe [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--any |strings |type for any matching (names or default), where type is client-id or client-id-prefix (repeatable).
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc]:
```suggestion
|--any |strings |Type for any matching (names or default), where type is `client-id` or `client-id-prefix` (repeatable).
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+|--default |strings |type for default matching, where type is client-id or client-id-prefix (repeatable).
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc]:
```suggestion
|--default |strings |Type for default matching, where type is `client-id` or `client-id-prefix` (repeatable).
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+
+|--format |string |Output format. Possible values: `json`, `yaml`, `text`, `wide`, `help`. Default: `text`.
+
+|-h, --help |- |Help for describe.
+
+|--name |strings |type=name pair for exact name matching, where type is client-id or client-id-prefix (repeatable).
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc]:
```suggestion
|--name |strings |The `type=name` pair for exact name matching, where type is `client-id` or `client-id-prefix` (repeatable).
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+|--strict |- |whether matches are strict, if true, entities with unspecified entity types are excluded.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc]:
```suggestion
|--strict |- |Specifies whether matches are strict. If `true`, entities with unspecified entity types are excluded.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc]:
```suggestion
|--config |string |Redpanda or `rpk` config file. Default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas.adoc [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--format |string |Output format. Possible values: `json`, `yaml`, `text`, `wide`, `help`. Default: `text`.
+
+|-h, --help |- |Help for quotas.
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas.adoc]:
```suggestion
|--config |string |Redpanda or `rpk` configuration file. Default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc [hunk 15]
```diff
@@ -14,6 +14,13 @@ Available tests to run:
 ** Throughput test: 8192-bit messages
 *** Unique pairs of Redpanda nodes each act as a client and a server.
 *** The test pushes as much data over the wire, within the test parameters.
+* *Cloud tests*
+** Latency test: 1024-bit object.
+** Depending on cluster read/write permissions (xref:reference:properties/object-storage-properties.adoc#cloud_storage_enable_remote_read[`cloud_storage_enable_remote_read`], xref:reference:properties/object-storage-properties.adoc#cloud_storage_enable_remote_write[`cloud_storage_enable_remote_write`]), a series of cloud storage operations are performed:
+*** Upload an object to an S3 bucket.
+*** List objects in the bucket.
+*** Download an object from the bucket.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc]:
```suggestion
*** Download an object from the S3 bucket.
```

File: modules/reference/pages/rpk/rpk-registry/rpk-registry-schema-check-compatibility.adoc [hunk 16]
```diff
@@ -18,11 +18,11 @@ rpk registry schema check-compatibility SUBJECT [flags]
 |-h, --help |- |Help for check-compatibility.
 |--references |string |Comma-separated list of references (name:subject:version), or path to reference file.
 
-|--schema |string |Schema filepath to check, must be `.avro` or `.proto`.
+|--schema |string |Schema filepath to check, must be `.avro`, `.json` or `.proto`.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-registry/rpk-registry-schema-check-compatibility.adoc]:
```suggestion
|--schema |string |Schema file path to check. Must be `.avro`, `.json` or `.proto`.
```

File: modules/reference/pages/rpk/rpk-registry/rpk-registry-schema-check-compatibility.adoc [hunk 17]
```diff
@@ -18,11 +18,11 @@ rpk registry schema check-compatibility SUBJECT [flags]
...
 
 |--schema-version |string |Schema version to check compatibility with (`latest`, `0`, `1`...).
 
-|--type |string |Schema type (`avro`,`protobuf`); overrides schema file extension.
+|--type |string |Schema type (`avro`, `json`, `protobuf`); overrides schema file extension.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-registry/rpk-registry-schema-check-compatibility.adoc]:
```suggestion
|--type |string |Schema type (`avro`, `json`, `protobuf`). Overrides schema file extension.
```

File: modules/reference/pages/rpk/rpk-registry/rpk-registry-schema-get.adoc [hunk 18]
```diff
@@ -33,11 +33,11 @@ rpk registry schema get [SUBJECT] [flags]
 
 |--print-schema |- |Prints the schema in JSON format.
 
-|--schema |string |Schema filepath to upload, must be `.avro`, `.avsc`, or `.proto`.
+|--schema |string |Schema filepath to upload, must be `.avro`, `.avsc`, `json`, or `.proto`.
 
 |--schema-version |string |Schema version to check compatibility with (`latest`, `0`, `1`...).
 
-|--type |string |Schema type of the file used to lookup (avro, protobuf); overrides schema file extension.
+|--type |string |Schema type of the file used to lookup (`avro`, `json`, `protobuf`); overrides schema file extension.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-registry/rpk-registry-schema-get.adoc]:
```suggestion
|--type |string |Schema type of the file used to lookup (`avro`, `json`, `protobuf`). Overrides schema file extension.

```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc [hunk 19]
```diff
@@ -5,26 +5,17 @@ Deploy a data transform.
 
 When run in the same directory as a `transform.yaml`, this reads the configuration file, then looks for a `.wasm` file with the same name as your project. If the input and output topics are specified in the configuration file, those are used. Otherwise, the topics can be specified on the command line using the `--input-topic` and `--output-topic` flags.
 
-== Example
+Environment variables can be specified for the transform using the `--var` flag, these are separated by an equals for example: `--var=KEY=VALUE`. The `--var` flag can be repeated to specify multiple variables.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc]:
```suggestion
You can specify environment variables for the transform using the `--var` flag. Variables are separated by an equal sign. For example: `--var=KEY=VALUE`. The `--var` flag can be repeated to specify multiple variables.

```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc [hunk 20]
```diff
@@ -5,26 +5,17 @@ Deploy a data transform.
 
 When run in the same directory as a `transform.yaml`, this reads the configuration file, then looks for a `.wasm` file with the same name as your project. If the input and output topics are specified in the configuration file, those are used. Otherwise, the topics can be specified on the command line using the `--input-topic` and `--output-topic` flags.
 
-== Example
+Environment variables can be specified for the transform using the `--var` flag, these are separated by an equals for example: `--var=KEY=VALUE`. The `--var` flag can be repeated to specify multiple variables.
 
-To deploy Wasm files directly without a `transform.yaml` file:
+The `--from-offset` flag can be used to specify where on the input topic the transform should begin processing. Expressed as:
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc]:
```suggestion
You can specify the `--from-offset` flag to identify where on the input topic the transform should begin processing. Expressed as:

```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc [hunk 21]
```diff
@@ -5,26 +5,17 @@ Deploy a data transform.
...
+- `+N` - Begin reading N records from the start of each input partition
 
-The --var flag can be repeated to specify multiple variables like so:
+- `-N` - Begin reading N records prior to the end of each input partition
 
-[,bash]
-----
-rpk transform deploy --var FOO=BAR --var FIZZ=BUZZ
-----
+Note that the broker will only respect from-offset on the first deploy for a given transform. Re-deploying the transform will cause processing to pick up at the last committed offset. Recall that this state is maintained until the transform is deleted.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc]:
```suggestion
Note that the broker will only respect `--from-offset` on the first deploy for a given transform. Re-deploying the transform will cause processing to pick up at the last committed offset. This state is maintained until the transform is deleted.

```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc [hunk 22]
```diff
@@ -0,0 +1,34 @@
...
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|-h, --help |- |Help for pause.
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc]:
```suggestion
|--config |string |Redpanda or `rpk` configuration file. Default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc [hunk 23]
```diff
@@ -0,0 +1,34 @@
...
+
+|-X, --config-opt |stringArray |Override rpk configuration settings; '-X help' for detail or '-X list' for terser detail.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc]:
```suggestion
|-X, --config-opt |stringArray |Override rpk configuration settings. Use '-X help' for detail or '-X list' for terser detail.
```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc [hunk 24]
```diff
@@ -0,0 +1,34 @@
...
+
+|--profile |string |rpk profile to use.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc]:
```suggestion
|--profile |string |The rpk profile to use.
```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc [hunk 25]
```diff
@@ -0,0 +1,32 @@
...
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|-h, --help |- |Help for resume.
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc]:
```suggestion
|--config |string |Redpanda or `rpk` configuration file. Default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc [hunk 26]
```diff
@@ -0,0 +1,32 @@
...
+
+|-X, --config-opt |stringArray |Override rpk configuration settings; '-X help' for detail or '-X list' for terser detail.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc]:
```suggestion
|-X, --config-opt |stringArray |Override rpk configuration settings. Use '-X help' for detail or '-X list' for terser detail.
```

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc [hunk 27]
```diff
@@ -0,0 +1,32 @@
...
+
+|--profile |string |rpk profile to use.
```
Comment (Feediver1) [inline modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc]:
```suggestion
|--profile |string |The rpk profile to use.
```

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc (hunk 15 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc]: Is this feature S3 specific ? @r-vasquez

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc (hunk 6 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc (hunk 13 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas.adoc (hunk 14 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc (hunk 22 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc (hunk 23 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc (hunk 24 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc (hunk 26 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc (hunk 27 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

File: modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc (hunk 25 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-transform/rpk-transform-resume.adoc]: Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.

**Review by Feediver1 — APPROVED** [review]

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc (hunk 15 above)
Comment (r-vasquez) [reply modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc]: I confirmed with Core that this feature accepts either ABS or S3, so we need to remove any reference to `S3` in this new paragraph.

I'll make sure to bring this back to the rpk code.

**Review by r-vasquez — APPROVED** [review]

File: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc (hunk 15 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc]: Changed from s3 bucket to object storage

**Pull request merged**